### PR TITLE
16 migrate to java 11

### PR DESCRIPTION
**Pull request by kostobog**
Tests pass.

### CODE REVIEW

File: src/test/java/og_spipes/rest/ScriptControllerTest.java [hunk 1]
```diff
@@ -57,7 +58,9 @@ public void init() throws Exception {
         }
         FileUtils.copyDirectory(new File("src/test/resources/scripts_test/sample/hello-world"), scriptsHomeTmp);
         FileUtils.copyFileToDirectory(new File("src/test/resources/SHACL/rule-test-cases/data-without-label.ttl"), scriptsHomeTmp);
-        mapper.registerModule(new JsonLdModule());
+        JsonLdModule module = new JsonLdModule();
```
Comment (blcham) [inline src/test/java/og_spipes/rest/ScriptControllerTest.java:61]: those 3 lines look like should be wrapped into a single method within some kind of "test util" class, what do you think?

File: src/test/java/og_spipes/rest/ViewControllerTest.java [hunk 2]
```diff
@@ -54,7 +55,9 @@ public static void beforeAll() throws IOException {
         conn.add(new File("src/test/resources/rdf4j_source/repositories/rdf4j_export"), null, RDFFormat.TURTLE);
         conn.close();
         sesameRepo.shutDown();
-        mapper.registerModule(new JsonLdModule());
+        JsonLdModule module = new JsonLdModule();
```
Comment (blcham) [inline src/test/java/og_spipes/rest/ViewControllerTest.java:58]: same lines as before -- see my comment above.

**Review by blcham — APPROVED** [review]
Good job, see my comment and either fix and merge or only merge.

**Comment by blcham** [comment]
Docker build failed:
image

Thus I upgraded Dockerfile to execute maven with java 11 (see last commit), but no i get test failure:
image
